data/mapcss-mapping.csv

```diff
@@ -375,14 +394,17 @@ piste:type|downhill|advanced;[piste:type=downhill][piste:difficulty=advanced];;n
 sport|shooting;348;
 place|country;349;
 highway|path|alpine_hiking;[highway=path][sac_scale=alpine_hiking];;name;int_name;350;
-deprecated|deprecated;351;x
+# -branch includes ordinary railway=rail w/o any specific tags like usage= or service=.
+# Using service= together with usage=branch is a mismapping, but people do it, so exclude explicitly.
+railway|rail|branch;[railway=rail][usage=branch][!highspeed][!service],[railway=rail][!usage][!service][!highspeed];;name;int_name;351;
```

Comment:
So put exact this line into 28, former railway=rail.
And branch=bridge/tunnel into former rail=bridge/tunnel.

Reply:
but then `railway=rail usage/service=rare_unsupported_value` combinations will not be matched by any line...
and we need to show users a railway line there

please take a look into matching cases I put into unit tests..

Reply:
BTW, also you can delete old railway=rail from styles with this change.